packages/user/src/assets/css/signup.css

```diff
@@ -2,7 +2,7 @@
   justify-items: center;
 }
 
-.signup .links {
+.signup .signup-wrapper .links {
   display: flex;
   justify-content: space-between;
   margin-top: 1rem;
```

Comment:
Do we still need this css here? This should be the default and should be in signup-wrapper css. Same comment for login as well.